packages/data-point/examples/entity-request-options-override.js

```diff
@@ -0,0 +1,25 @@
+const dataPoint = require('../').create()
+const mockRequest = require('./entity-request-basic.mock')
+
+dataPoint.addEntities({
+  'request:getLuke': {
+    url: 'https://swapi.co/api/people/1/'
+  }
```

Comment:
I suggest adding this to the `request:getLuke` definition:
```js
params: {
  inspect: false
}
```
then, above line 17, have a comment saying that the params defined there will override the param defined in the entity definition. I think that will make it even more clear how the override is working.